modules/ROOT/pages/install-a-bonita-bpm-cluster.adoc

```diff
@@ -162,11 +162,11 @@ In this case you already have a Bonita Platform running as single node installat
 
 === Update the configuration in database
 
-Some properties of the Bonita Platform needs to be changed, through  xref:bonita-bpm-platform-setup.adoc[Bonita platform setup tool], in order to make your installation work as a cluster node.
+Some properties of the Bonita Platform needs to be changed, through  xref:bonita-platform-setup.adoc[Bonita platform setup tool], in order to make your installation work as a cluster node.
 
 * Download Bonita xref:tomcat-bundle.adoc[Tomcat bundle], that contains the platform setup tool, and unzip it at some place of your choice.
```

Comment:
at some place of your choice. => in your chosen folder. / in a folder of your choice.

Reply:
Done